mkdocs.yml

```diff
@@ -158,11 +159,19 @@ nav:
     - III - Functional Workflow: 'tutorials/basic/Tutorial-III-Functional-Workflow.ipynb'
     - IV - Customization: 'tutorials/basic/Tutorial-IV-Customization.ipynb'
   - Advanced:
-    - Equation Discovery: 'tutorials/Equation Discovery.ipynb'
+    - Equation Discovery: 'theorist/Equation Discovery.ipynb'
     - Experimentalists: 'tutorials/Experimentalist.ipynb'
```

Comment:
this isn't really an advanced tutorial, is it?

Reply:
Yes, it was in the advanced tutorials before, but was a broken link. I just changed the path. 

BTW: There is another tutorial file in the advanced folder called "Theorist.ipynb" that we don't use anywhere right now. I was not sure about that one either.

Reply:
We should also mark the `theorist/Equation Discovery.ipynb` file as requiring python3.10, because it uses the `case-match` statement which is first available then. (Or we should use the old `if-elif-else` syntax instead.) That would be a different PR though.